htdocs/commande/card.php

```diff
@@ -216,6 +216,8 @@
 		$result = $object->deleteline($user, $lineid);
 		if ($result > 0)
 		{
+			// reorder lines
+			$object->line_order(true);
```

Comment:
After a deletion, sort sort criteria should remains good. 
Why do you need this ?

Reply:
Hi @eldy, it's about following PR : https://github.com/Dolibarr/dolibarr/pull/19393
For this feature to work properly, we must have no break in lines rank numerotation.
Without "$object->line_order(true);" we can get rank list like 1,2,3,5,6,7 after line 4 has been deleted, then for my feature i always need to get rank list like 1,2,3,4,5,6 after line 4 deletion.

Reply:
And to be more precise, i just added the same code that is is already present in supplier propsal and propal cards !

![image](https://user-images.githubusercontent.com/5585819/142606407-ec633166-b691-4018-936d-4a425f043afa.png)